src/Sylius/Bundle/ApiBundle/spec/CommandHandler/Checkout/CompleteOrderHandlerSpec.php

```diff
@@ -160,6 +164,30 @@ function it_throws_an_exception_if_order_does_not_exist(
         ;
     }
 
+    function it_throws_an_exception_if_order_total_has_changed(
+        OrderRepositoryInterface $orderRepository,
+        StateMachineInterface $stateMachine,
+        OrderInterface $order,
+        FactoryInterface $stateMachineFactory,
+        CustomerInterface $customer,
+        OrderPromotionsIntegrityCheckerInterface $orderPromotionsIntegrityChecker,
```

Comment:
Afaik, the order of the arguments should first reflect the constructor of the class

```suggestion
        OrderRepositoryInterface $orderRepository,
        FactoryInterface $stateMachineFactory,
        OrderPromotionsIntegrityCheckerInterface $orderPromotionsIntegrityChecker,
        StateMachineInterface $stateMachine,
        OrderInterface $order,
        CustomerInterface $customer,

```